installer/picard-setup.nsi.in

```diff
@@ -142,6 +143,11 @@ Section "!$(SectionRequired)" required
     ; Write the installation path into the registry
     WriteRegStr HKLM "Software\${PRODUCT_PUBLISHER}\${PRODUCT_NAME}" "InstallDir" "$INSTDIR"
 
+    ; Register a custom protocol handler
+    WriteRegStr HKCR "${PRODUCT_CUSTOM_PROTOCOL}" "URL Protocol" ""
+    WriteRegStr HKCR "${PRODUCT_CUSTOM_PROTOCOL}" "DefaultIcon" "$INSTDIR\picard.exe,1"
+    WriteRegStr HKCR "${PRODUCT_CUSTOM_PROTOCOL}\shell\open" "command" "$INSTDIR\picard.exe %%1"
+
     ; Create uninstaller
```

Comment:
TODO: When Picard is uninstalled, those keys should be removed.

Reply:
I added a removal in the uninstall section